scripts/circleci/build.sh

```diff
@@ -2,6 +2,11 @@
 		
 set -e		
 
+# Update the local size measurements to the master version
+# so that the size diff printed at the end of the build is
+# accurate.
+curl -o scripts/rollup/results.json http://react.zpao.com/builds/master/latest/results.json
```

Comment:
I hesitate to bring it up, but should we move this to reactjs.org?

Not that, like, @zpao will hold our build ransom or anything :)

Reply:
Yeah maybe :-) I don't care strongly.

Reply:
Yeah, no worries. We need to figure out static hosting for the fixtures too. Maybe we can figure this out when/if we deal with that.